db/schema.rb

```diff
@@ -46,6 +47,16 @@
     t.index ["link"], name: "index_recommended_links_on_link", unique: true
   end
 
+  create_table "serving_configs", charset: "utf8mb3", force: :cascade do |t|
+    t.string "display_name", null: false, comment: "A human-readable name"
+    t.string "description", null: false, comment: "A description of this serving config's purpose"
+    t.string "remote_resource_id", null: false, comment: "The ID of this serving config on Discovery Engine"
+    t.boolean "users_can_assign_controls", null: false, comment: "Whether or not users can assign controls to this serving config in the UI"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["remote_resource_id"], name: "index_serving_configs_on_remote_resource_id", unique: true
+  end
+
```

Comment:
Where has this come from? I'm guessing that you have another change on the go in a different branch?

Reply:
😭